controllers/module_reconciler.go

```diff
@@ -367,12 +381,27 @@ func (mrh *moduleReconcilerHelper) handleDriverContainer(ctx context.Context,
 	})
 
 	if err == nil {
-		logger.Info("Reconciled Driver Container", "name", ds.Name, "result", opRes)
+		logger.Info("Reconciled Module Loader Container", "name", ds.Name, "result", opRes)
 	}
 
 	return err
 }
 
+func (mrh *moduleReconcilerHelper) handleModuleOrderConfigMap(ctx context.Context, mld *api.ModuleLoaderData) error {
```

Comment:
Do we need a ConfigMap? The config could be a Pod annotation that we could mount using the [downward API](https://kubernetes.io/docs/concepts/workloads/pods/downward-api/).

Reply:
That is not an optimal solution, since you cannot map each label/annotation individually. So you will get a file with label=<value>. I think it is easier to do it with ConfigMap, and will be easier to debug. The only change is that we create one configmap per Module.

Reply:
Fixed